packages/required/index.ts

```diff
@@ -13,8 +13,8 @@ interface RequiredDescriptor extends Descriptor {
 interface StampMethodRequired {
   (this: Stamp, settings: Composable): Stamp;
 }
-export const required: StampMethodRequired = function required(settings): Stamp {
-  const localStamp = this?.compose ? this : Required;
+export const required: StampMethodRequired = function required(this: Stamp, settings): Stamp {
+  const localStamp = this ? this : Required;
```

Comment:
This is not yet covered by a unit test, unfortunately, but `this?.compose` shouldn't have been changed. It's important to have.
Alternatively, we can change the line to:
```js
const localStamp = isStamp(this) ? this : Required;
```

Reply:
The compile error claims that this?.compose never fails. If this should not change then there is something else that needs to change.

Reply:
That will probably keep the compiler happy

Reply:
isStamp does indeed solve the issue. Thanks!